bndtools.core/src/bndtools/refactor/PkgRenameParticipant.java

```diff
@@ -66,8 +65,9 @@ protected boolean initialize(Object element) {
 		sb.append("Bndtools: rename package '");
 		sb.append(pkgFragment.getElementName());
 		sb.append("' ");
-		if (((RenamePackageProcessor) this.getProcessor()).getRenameSubpackages())
-			sb.append("and subpackages ");
+		// if (((RenamePackageProcessor)
+		// this.getProcessor()).getRenameSubpackages())
+		// sb.append("and subpackages ");
```

Comment:
I think just delete this rather than leave it commented out.

Reply:
Agree. We can always try to resurrect from Git and commented code can be extremely confusing.